jwst/datamodels/sossmodel.py

```diff
@@ -0,0 +1,40 @@
+from .model_base import JwstDataModel
+
+__all__ = ['SossModel']
+
+
+class SossModel(JwstDataModel):
```

Comment:
I wonder if we could use a slightly less generic class name for this model. Perhaps something like `SossExtractModel`, just so it's more obvious that it's associated with stuff used as part of the extraction?